airflow/config_templates/config.yml

```diff
@@ -210,6 +210,15 @@
       example: ~
       default: "False"
       see_also: "https://docs.python.org/3/library/pickle.html#comparison-with-json"
+    - name: allowed_deserialization_classes
```

Comment:
This only affects xcom right now. Is the goal to eventually make it affect serialised dags too, or should we change the config name?

Reply:
I'd like to make it the generic serializer for Airflow, so eventually also DAGs (pending feasibility)